src/components/icon/icon.spec.js

```diff
@@ -419,6 +419,7 @@ describe('MdIcon service', function() {
     $scope = $rootScope;
 
     $templateCache.put('android.svg'    , '<svg><g id="android"></g></svg>');
+    $templateCache.put('angular-logo.svg','<svg><g id="angular"></g><defs><filter id="shadow"></filter></defs><path filter="url(#shadow)"></path></svg>');
```

Comment:
We need to test a more complex SVG here. Ideally with a couple of different ids.